src/pykeen/ablation/ablation.py

```diff
@@ -293,12 +429,24 @@ def prepare_ablation(  # noqa:C901
         model_to_model_kwargs = {}
     if not model_to_model_kwargs_ranges:
         model_to_model_kwargs_ranges = {}
-    if not model_to_trainer_to_training_kwargs:
-        model_to_trainer_to_training_kwargs = {}
-    if not model_to_trainer_to_training_kwargs_ranges:
-        model_to_trainer_to_training_kwargs_ranges = {}
-    if not ablation_config:
-        ablation_config = {}
+    if not model_to_loss_to_loss_kwargs:
```

Comment:
I don't think all of these default assignments are necessary anymore since 5f2cd86b

Reply:
Nice! Adapted in 4805926505253efe691858b9848998705c99a269.